runtime/manager_test.go

```diff
@@ -4243,7 +4245,8 @@ func TestNewManagerFromReaders(t *testing.T) {
 			args: args{
 				configReader: strings.NewReader(string(`{
 					"p2p": {
-					  "consensus_port": 8080,
+					  "hostname": "node1.consensus",
+					  "port": 8080,
```

Comment:
Might be worth doing a global search for "8080" across the whole codebase.

Reply:
For sure, I did this but maybe things have changed in the meantime. Will double-check. Good call to look for the other port numbers.